Evaluation:
    Indirect_Chip:
      chip_target_fraction:
        min: 0.0
        max: 100.0
        default: 5.0/10.0
        type: "double"
        description: Adjusts how far between ball and target the robot will chip
      chip_power_bounce_threshold:
        min: 0.0
        max: 5.0
        default: 7.5/10.0
        type: "double"
        description: Maximum fraction of distance between chipper and target the first bounce should be, so ball is rolling when it reaches the target
      max_chip_power:
        min: 0.0
        max: 100.0
        default: 8.0
        type: "double"
        description: Maximum power the robot can chip the ball at without malfunctions
      chip_target_area_inset:
        min: 0.0
        max: 100.0
        default: 0.3
        type: "double"
        description: Closest distance to edge of field that the robot could chip and chase to
      min_chip_tri_area:
        min: 0.0
        max: 100.0
        default: 0.5
        type: "double"
        description: Minimum area of chip target triangle
      min_chip_tri_edge_len:
        min: 0.0
        max: 100.0
        default: 0.8
        type: "double"
        description: Minimum edge length of chip target triangle
      min_chip_tri_edge_angle:
        min: 0.0
        max: 180.0
        default: 20
        type: "double"
        description: Minimum angle in degrees between chip triangle edges
      chip_cherry_power_downscale:
        min: 0.0
        max: 100.0
        default: 0.85
        type: "double"
        description: Percentage of distance to center of triangle to return as target
